### PR TITLE
Prime Factorisation

### PR DESCRIPTION
**Pull request by Paradact**
Added prime factorisation problems

### CODE REVIEW

File: mathgenerator/mathgen.py [hunk 1]
```diff
@@ -216,6 +216,21 @@ def multiplyIntToMatrix22(maxMatrixVal = 10, maxRes = 100):
...
+    i = 2
+    factors = []
+    while i*i <= a:
+        if a % i:
+            i += 1
+        else:
+            a //= i
+            factors.append(i)
+    if a > 1:
+        factors.append(n)
```
Comment (lukew3) [inline mathgenerator/mathgen.py:385]: This variable n is not defined

**Review by lukew3 — CHANGES_REQUESTED** [review]

**Review by lukew3 — APPROVED** [review]

**Comment by lukew3** [comment]
When I tested your code I found that it rarely worked. Here's 10 tests:
```
('Find prime factors of 7', '[2, 2, 2, 2, 2, 7]')
('Find prime factors of 19', '[2, 5, 19]')
('Find prime factors of 13', '[2, 2, 5, 13]')
('Find prime factors of 3', '[2, 2, 2, 2, 2, 3]')
('Find prime factors of 761', '[761]')
('Find prime factors of 43', '[17, 43]')
('Find prime factors of 613', '[613]')
('Find prime factors of 47', '[3, 47]')
('Find prime factors of 349', '[2, 349]')
('Find prime factors of 367', '[367]')
```
Be sure to test your code before submitting a pr. You can include
```
for i in range(0, 10):
    print(primeFactors())
```
in the last line of your code to test it multiple times.

**Review by lukew3 — CHANGES_REQUESTED** [review]
Test your code and fix errors.

**Review by lukew3 — APPROVED** [review]

**Comment by lukew3** [comment]
Good work

**Comment by Paradact** [comment]
Thank you for your feedback, I have adjusted the problem since it was previously being adjusted itself in the algorithm, as well as adjusted the difficulty to be between 1 and 200

**Comment by lukew3** [comment]
Thank you, it looks perfect now.

**Pull request merged**